### ISSUE TITLE
User Export / Import utility

### ISSUE THREAD

**Issue by prepperz** [PHPfox-Official/phpfox-v4-feature-requests#748]
This is very useful if a rollback is needed, but you don't want to lose any current users.

There are certain situations where you get new users for today, but for some critical reasons you need to restore the site from backup from a few days ago.

It would be very useful to be able to selectively export certain users and then import those users into the newly restored site.

Other uses cases may also apply to the following
--------------------------------------------------------
There are times when upgrades have issues. It would save many customers a lot of time and aggravation if there is an option to use utility to import all user data into a new install instead upgrades for the following reason: https://community.phpfox.com/forum/thread/57593/post-4-6-to-4-7-upgrade-database-table-inconsistencies-in-db-comparisons-to/

This is not to mention that sometimes the upgrade script / process that would kill sites due to whatever reasons that seems to be a regular thing with phpfox upgrades.

**Comment by harrison05** [comment]
Do you want a new feature "Import Users"?

**Comment by prepperz** [comment]
Yes. Importing (as well as Exporting) selected users and all their data (images, videos,etc...) can be very useful utility.
